onnxruntime/python/tools/quantization/quantize.py

```diff
@@ -888,18 +888,18 @@ def _handle_other_ops(self, node, new_nodes_list):
 
     def _handle_activation_ops(self, node, new_node_list):
         '''
-        Checks whether the give activation op can be removed from the graph. When mode is QLinearOps, 
-        the output quatization params are calculated based on outputs from activation nodes, 
+        Checks whether the give activation op can be removed from the graph. When mode is QLinearOps,
```

Comment:
I formatted the quantize.py with yapf. You may have to resolve the conflict.

Reply:
Thanks, I merged master again and reapplied some typo fixes.